metadata-service/configuration/src/main/resources/application.yml

```diff
@@ -346,6 +346,7 @@ featureFlags:
   preProcessHooks:
     uiEnabled: ${PRE_PROCESS_HOOKS_UI_ENABLED:true} # Circumvents Kafka for processing index updates for UI changes sourced from GraphQL to avoid processing delays
   showAcrylInfo: ${SHOW_ACRYL_INFO:false} # Show different CTAs within DataHub around moving to Managed DataHub. Set to true for the demo site.
+  ermodelrelationFeatureEnabled: ${ERMODELRELATION_FEATURE_ENABLED:false} # Enable Join Tables Feature and show within Dataset view as Relations
```

Comment:
can we update the casing and name here to be `erModelRelationshipFeatureEnabled` (and all the places referencing this) and then change the env variable to be `ER_MODEL_RELATIONSHIP_FEATURE_ENABLED`